runtime.txt

```diff
@@ -0,0 +1 @@
+2.7
```

Comment:
@wlonk Is this a valid syntax? It's not what I'm used to, but seems to be what Netlify is expecting:

> /runtime.txt may list a Python version to use from the list of versions preinstalled in your site’s selected build image. Note that the syntax for this file is JUST the version number: x.y, with no trailing newline.

https://www.netlify.com/docs/build-settings/#python

Reply:
Yup, looks correct. `runtime.txt` ain't any kind of a standard; Heroku expects one set of things there that we use, but Netlify has a totally different set of expectations.